casepro/pods/tests/test_base.py

```diff
@@ -0,0 +1,22 @@
+import json
```

Comment:
We're using Python 2.x (as I assume you guys are? Never tested this thing in Python 3) so you'll want to start all your files with `from __future__ import unicode_literals`